api/user/teacherUtil.js

```diff
@@ -12,9 +12,41 @@ const {
   customMessageError,
 } = require("../util/error_handling");
 
-const teacherSchemaNew = Joi.object({
+const teacherSchemaNew = Joi.object({// TODO beginner: finish defining schema - DONE
```

Comment:
You have added `email`, `first_name`, etc which are already on the User model.

Please reference the DB migration file here:
external/db/migrations/20211006204848_teacher_base.js

Note: the language code will ideally be checked against the list of language codes in the ISO-639-1 dependency (i.e. the input should exist in that list of language codes). But you can just use a simple list like the following if you'd like.

```
["JA", "EN"]
```